js/main.js

```diff
@@ -1,30 +1,86 @@
+      $img.setAttribute('id', teamID);
+    } else if (teamName === 'Los Angeles Kings') {
       $img.setAttribute('src', 'https://upload.wikimedia.org/wikipedia/en/thumb/6/63/Los_Angeles_Kings_logo.svg/1200px-Los_Angeles_Kings_logo.svg.png');
-    } else if (xhr.response.records[3].teamRecords[i].team.name === 'Edmonton Oilers') {
+      $img.setAttribute('id', teamID);
+    } else if (teamName === 'Edmonton Oilers') {
       $img.setAttribute('src', 'https://upload.wikimedia.org/wikipedia/en/thumb/4/4d/Logo_Edmonton_Oilers.svg/1200px-Logo_Edmonton_Oilers.svg.png');
-    } else if (xhr.response.records[3].teamRecords[i].team.name === 'Seattle Kraken') {
+      $img.setAttribute('id', teamID);
+    } else if (teamName === 'Seattle Kraken') {
       $img.setAttribute('src', 'https://upload.wikimedia.org/wikipedia/en/thumb/4/48/Seattle_Kraken_official_logo.svg/1200px-Seattle_Kraken_official_logo.svg.png');
-    } else if (xhr.response.records[3].teamRecords[i].team.name === 'Calgary Flames') {
+      $img.setAttribute('id', teamID);
+    } else if (teamName === 'Calgary Flames') {
       $img.setAttribute('src', 'https://upload.wikimedia.org/wikipedia/en/thumb/6/61/Calgary_Flames_logo.svg/1200px-Calgary_Flames_logo.svg.png');
-    } else if (xhr.response.records[3].teamRecords[i].team.name === 'Vancouver Canucks') {
+      $img.setAttribute('id', teamID);
+    } else if (teamName === 'Vancouver Canucks') {
       $img.setAttribute('src', 'https://upload.wikimedia.org/wikipedia/en/thumb/3/3a/Vancouver_Canucks_logo.svg/1200px-Vancouver_Canucks_logo.svg.png');
-    } else if (xhr.response.records[3].teamRecords[i].team.name === 'San Jose Sharks') {
+      $img.setAttribute('id', teamID);
+    } else if (teamName === 'San Jose Sharks') {
       $img.setAttribute('src', 'https://upload.wikimedia.org/wikipedia/en/thumb/3/37/SanJoseSharksLogo.svg/1200px-SanJoseSharksLogo.svg.png');
-    } else if (xhr.response.records[3].teamRecords[i].team.name === 'Anaheim Ducks') {
+      $img.setAttribute('id', teamID);
+    } else if (teamName === 'Anaheim Ducks') {
       $img.setAttribute('src', 'https://upload.wikimedia.org/wikipedia/en/thumb/7/72/Anaheim_Ducks.svg/1200px-Anaheim_Ducks.svg.png');
+      $img.setAttribute('id', teamID);
     }
     $li.appendChild($img);
     $ol.appendChild($li);
   }
 });
 xhr.send();
+
+var $imagesContainer = document.querySelector('.listed-images-container');
+$imagesContainer.addEventListener('click', function (event) {
+  if (event.target.tagName !== 'IMG') {
+    return;
+  }
+  $rankingsContainer.classList.add('hidden');
+  $tableContentContainer.classList.remove('hidden');
+  var teamID = event.target.id;
+  var xhr1 = new XMLHttpRequest();
+  xhr1.open('GET', 'https://statsapi.web.nhl.com/api/v1/teams/' + teamID + '/roster?season=20222023');
+  xhr1.responseType = 'json';
+  xhr1.addEventListener('load', function () {
+    var $body = document.querySelector('tbody');
+    if ($body) {
+      $body.remove();
```

Comment:
I would consider renaming this. This can be confusing, because it could also mean you're referring to the `body` of the HTML document. Maybe something like `$oldTableBody` or `$existingTableBody`